script/tool/lib/src/lint_podspecs_command.dart

```diff
@@ -132,6 +132,7 @@ class LintPodspecsCommand extends PluginCommand {
       podspecPath,
       '--configuration=Debug', // Release targets unsupported arm64 simulators. Use Debug to only build against targeted x86_64 simulator devices.
       '--skip-tests',
+      '--use-modular-headers', // Flutter sets use_modular_headers! in its templates.
```

Comment:
> // Flutter sets use_modular_headers! in its templates.

It does that for Swift only.  I _think_ by adding this, `--use-libraries` will no longer test the plugins with static libraries, which is the default for Flutter Objective-C projects.  The intention of running this once with and once without `--use-libraries` is to test both.

https://github.com/flutter/plugins/blob/15cedbc3f0c9f0101e82b791e8e382e03ef284e0/script/tool/lib/src/lint_podspecs_command.dart#L110-L120

Instead, you may want to somehow detect a plugin is Swift (does the podspec contains `swift_version`) and skip the  `libraryLint: true` run.

Reply:
I was confusing this with `use_frameworks!`.  I'm vaguely remembering this now https://github.com/flutter/flutter/issues/41007.

If it passes with `--use-modular-headers` let's keep it as you have it.